spec/builtin-bash.test.sh

```diff
@@ -37,3 +37,16 @@ echo "help=$?"
 cat $TMP/err.txt | grep -i 'no help topics' >/dev/null
 echo "grep=$?"
 ## stdout-json: "help=1\ngrep=0\n"
+
+#### type -p builtin -> file
+type -p mv tar grep
+## stdout-json: "/bin/mv\n/bin/tar\n/bin/grep\n"
```

Comment:
Please use the newer format here:

```
## STDOUT:
/bin/mv
...
## END
```

For awhile, we didn't have anything but the JSON format, but multiline is more readable for many cases.  I tend to change those when touching older files.